javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1177,9 +1177,15 @@ class Options {
       expiry = Math.floor(date.getTime() / 1000)
     }
 
-    if (sameSite && sameSite !== 'Strict' && sameSite !== 'Lax') {
+    if (sameSite && !['Strict', 'Lax', 'None'].includes(sameSite)) {
       throw new error.InvalidArgumentError(
-        `Invalid sameSite cookie value '${sameSite}'. It should be either "Lax" (or) "Strict" `
+        `Invalid sameSite cookie value '${sameSite}'. It should be one of "Lax", "Strict" or "None"`
+      )
+    }
+
+    if (sameSite === 'None' && !secure) {
+      throw new error.InvalidArgumentError(
+        `Invalid cookie configuration: SameSite=None must be Secure`
```

Comment:
It helps further understanding that `None` is a string and we keep a consistent sameSite formatting
```suggestion
        'Invalid cookie configuration: sameSite="None" must be Secure'
```

Reply:
I was following the cookie header format (e.g. `SameSite=None; Secure; `). 
I'll change if it's cleaner. (can't commit your suggestion or else the test will break.)